src/api/test-helpers/create-multiple-accounts.ts

```diff
@@ -0,0 +1,21 @@
+import accounts from './accounts';
+import { Account, Context } from '../types';
+
+/**
+ * createMultipleAccounts
+ * Create multiple accounts for unit testing
+ * @param {Object} KeystoneJS context API
+ * @returns {Promise<Array<Account>>} Accounts array
+ */
+export const createMultipleAccounts = async (context: Context): Promise<Array<Account>> => {
+  const account1 = (await accounts.create({ context })) as Account;
```

Comment:
What's the reason for _not_ deleteing accounts in the first call here like the others?

Reply:
Could `accounts.create` have a default `deleteAccounts` param of false? Save passing `false` to it multiple times here.

Reply:
accounts.create is being used in 40+ other places and I think they need a delete to be set to default true

Reply:
Understood, makes sense as it is. This seems to be the only place where we need to _not_ delete accounts 👍